pkg/config/config.go

```diff
@@ -31,32 +31,40 @@ import (
 const (
 	// AgentWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogAgent controller.
 	agentWatchNamespaceEnvVar = "DD_AGENT_WATCH_NAMESPACE"
-	// SLOWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogSLO controller.
-	sloWatchNamespaceEnvVar = "DD_SLO_WATCH_NAMESPACE"
+	// DashboardWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogDashboard controller.
+	dashboardWatchNamespaceEnvVar = "DD_DASHBOARD_WATCH_NAMESPACE"
+	// GenericResourceWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogGenericResource controller.
+	genericResourceWatchNamespaceEnvVar = "DD_GENERIC_RESOURCE_WATCH_NAMESPACE"
 	// MonitorWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogMonitor controller.
 	monitorWatchNamespaceEnvVar = "DD_MONITOR_WATCH_NAMESPACE"
 	// ProfilesWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogAgentProfile controller.
 	profileWatchNamespaceEnvVar = "DD_AGENT_PROFILE_WATCH_NAMESPACE"
+	// SLOWatchNamespaceEnvVar is a comma-separated list of namespaces watched by the DatadogSLO controller.
+	sloWatchNamespaceEnvVar = "DD_SLO_WATCH_NAMESPACE"
```

Comment:
This is simply a re-ordering by alphabetical order, `DD_SLO_WATCH_NAMESPACE` was already present